agent/consul/auto_encrypt.go

```diff
@@ -116,27 +115,14 @@ func (c *Client) RequestAutoEncryptCerts(servers []string, defaultPort int, toke
 
 // resolveAddr is used to resolve the host into IPs, port, and error.
 // If no port is given, use the default
```

Comment:
This comment should be updated to remove references to ports, since resolveAddr just returns IPs now.